x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/elser/ElserMlNodeService.java

```diff
@@ -196,4 +198,9 @@ public String name() {
 
     @Override
     public void close() throws IOException {}
+
+    @Override
+    public TransportVersion getMinimalSupportedVersion() {
+        return TransportVersions.ML_INFERENCE_TASK_SETTINGS_OPTIONAL_ADDED;
```

Comment:
Is this the right value for this? It seemed weird to set this to a value defined prior to this PR even though I didn't change anything in the logic of this service 🤔

Reply:
We talked about this offline, we want the most recent transport version that was added for the elser service. So I changed it to that.